camera_node/CameraProperty property
camera_node/CameraPropertyMenuEntry[] menu_entries